src/main/java/team/gif/robot/UI.java

```diff
@@ -12,5 +13,11 @@ public UI() {
         ShuffleboardTab shuffleboardTab = Shuffleboard.getTab("FRC2023");
 
         shuffleboardTab.add("Arm Ticks", arm.getTicks()); // displaying the ticks for the arm.
+
+        // Pigeon
+        shuffleboardTab.add("Bot Heading", (x) -> {
```

Comment:
For now this is fine but we don't want to use a new tab in the final build

Reply:
ok